### ISSUE TITLE
Bug: WWE 2008 to 2011 games graphics flickering.

### ISSUE THREAD

**Issue by Satan86** [RPCS3/rpcs3#12649]
### Quick summary

WWE games from 2008 to 2011 have flickering graphics. None of the accuracy settings I could think of worked to fix it and I tested as far back as January 2020 in case it was a regression - it was not.

### Details

https://user-images.githubusercontent.com/61447828/189733052-9aa1af4d-be96-4bbf-bd36-d2cd1a9ae1fd.mp4

### Attach a log file

[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/9550993/RPCS3.log.gz)

### Attach capture files for visual issues

https://drive.google.com/file/d/1ST_r23p3uk6wGBg3fgOurbcom72XCG_C/view?usp=sharing

### System configuration

Ryzen 5 5600x, GTX 1050 Ti driver 516.59, Windows 11.

**Comment by vaishnav67** [comment]
Still occurring on RPCS3 v0.0.24-14176-bd1fb864 Alpha | master but it's like 5% better
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/9638246/RPCS3.log.gz)

https://user-images.githubusercontent.com/32519167/192088557-a2f3b3ca-9d7c-4710-b2ff-6f387500d79a.mp4

**System configuration**
OS: Windows 11
CPU: Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
GPU: Nvidia RTX 2060 Mobile (Driver version 516.94)

**Comment by generalvimal** [comment]
I am on RPCS3 Version: 0.0.29-15711-e26d4e17 Alpha | master

The issue still exists. Is the a fix for this?

System configuration
Windows 10
Intel I5 10th Gen
Nvidia GTX 1650 Mobile (Driver Version 546.01)

**Comment by Firas-Dhouibi** [comment]
RPCS3 V0.0.29 15811-d18f929f
yup, the issue still exists

**Comment by xMoJox12** [comment]
> Still occurring on RPCS3 v0.0.24-14176-bd1fb864 Alpha | master but it's like 5% better [RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/9638246/RPCS3.log.gz)
> 
>  WWESVR.2011.Graphics.mp4 
> **System configuration** OS: Windows 11 CPU: Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz GPU: Nvidia RTX 2060 Mobile (Driver version 516.94)

Same issue here with an Nvidia RTX 4060 and  AMD Ryzen 5 7600

**Comment by terminator1990t2** [comment]
same issue here with RCPS3 0.0.30-16061. I got a nividia rtx 3060 i7-8700 and have tried tweaking a few settings but nothings fixing it.

**Comment by AbdullahRizwan101** [comment]
Any fix for this ?

**Comment by azrulshah** [comment]
my game also same like this but sometime my wwesvr2008 stuck at main screen sometime loading screen, sometime can play the game all the way, can someone help?im using rog ally

**Comment by Mike-and-Ikes** [comment]
Hey sorry about the fat necro but I believe I have found a workaround to this particular problem.

In your config settings go to Advanced > GPU > Disable Vertex Cache

Make sure this is checked and it will eliminate 99% of the flickering you were talking about in your post. I've only tested this in SVR 2011 though on rpcs3 v0.035-17655-a1fa8dd7 Alpha.

When I went into road to wrestlemania mode in that particular game my graphics were practically exploding until I disabled the vertex cache. Now its very infrequent and it usually only shows up when I first load into the backstage area.

Hope this helps

**Comment by ElTioRata** [comment]
Thanks, disabling Vertex Cache works. The game still prone to crashes though (I got a "RSX: FIFO error: possible desync event (last cmd = 0xb)" after finishing a match).

**Comment by digant73** [comment]
@Satan86 is that still an issue?

**Comment by Satan86** [comment]
yeah still an issue, still fixed by disable vertex cache, but thats not a valid setting to close this

**Comment by digant73** [comment]
@Satan86 one quite recent PR improved/fixed vertex explosions on some games. any impact on these games?

**Comment by vaishnav67** [comment]
Edit: Nvm there's micro flickers occurring during matches but it is better than before for sure

I'm on RPCS3 Version: 0.0.41-19499-e32d19ff Alpha | master and it seems like the issue has been resolved! I think at this point the game might be fully playable but would need further testing
<img width="1919" height="1079" alt="Image" src="https://github.com/user-attachments/assets/7a2e5715-49a5-43ea-86d4-17e501e078ba" />

**Comment by Satan86** [comment]
Still an issue.
